scripts/generate_dump

```diff
@@ -642,6 +892,40 @@ get_asic_count() {
     echo `eval ${cmd} 2>&1`
 }
 
+###############################################################################
+# Get counter snapshot
+# Globals:
+#  None
+# Arguments:
+#  asic_name: Name of the asic vendor
+#  index: Index of counter snapshot
+# Returns:
+#  None
+###############################################################################
+save_counter_snapshot() {
+    local asic_name="$1"
+    local idx=$2
+    counter_t=$(date +'%d/%m/%Y %H:%M:%S:%6N')
+
+    save_cmd "echo $counter_t" "date.counter_$idx"
+    save_cmd "show interface counters" "interface.counters_$idx"
+    save_cmd "show queue counters" "queue.counters_$idx"
```

Comment:
use "save_cmd_all_ns" for "show queue counters", so that it works on multi ASIC platform

Reply:
Done